setup.py

```diff
@@ -70,17 +70,31 @@ def run(self):
     "dremio": ["pyarrow>=0.15.1"],
 }
 
+
 if sys.version_info < (3, 6):
     extras["dev"].remove("black")
 
 extras["all_extras"] = sum(extras.values(), [])
 
+# Extras for docker image builds to include for orchestration
+extras["all_orchestration_extras"] = [
+    "atlassian-python-api >= 2.0.1",
+    "azure-storage-blob >= 12.1.0, < 13.0",
+    "boto3 >= 1.9, < 2.0",
+    "kubernetes >= 9.0.0a1, <= 11.0.0b2",
+    "google-cloud-storage >= 1.13, < 2.0",
+    "python-gitlab >= 2.5.0, < 3.0",
+    "PyGithub >= 1.51, < 2.0",
+]
```

Comment:
Instead of duplicating these requirements here, we should build them programmatically out of a set of extras we want to include (similar to how `task_library_ci` is done). Would merge deps for each needed extra e.g. [`aws`, `github`, etc...].

Reply:
I considered that but we have extra dependencies in there for those names. e.g. azure has three, gcp/google has two, kubernetes has two, etc..

Reply:
Ah, hmmm. My main concern is different versions listed for each - perhaps assign the ones listed here as variables and then re-use them in both locations? I just don't want to have to bump the boto3 version in two spots.